src/domain/resource/card/ResourceCard.js

```diff
@@ -57,6 +58,33 @@ class ResourceCard extends React.Component {
     history.push(searchUtils.getSearchPageLink(filters));
   };
 
+  getAllowedLoginMethodsIcons = (t) => {
```

Comment:
This is almost identical to https://github.com/andersinno/varaamo/pull/55/files#diff-cd3d97b639549b796901a135232be5b009759c7622b72a5a0d6cbbfb06589cc8R44, would you be able to create one function for both cases so we don't duplicate code?

Reply:
I agree that this is very similar but I needed different classes for styling and the text also needs to go to different place. That is the reason why I wrote them separetely. A single function that needs different classes and text position would be complicated than this IMO.